formats/application/vnd/oada/isoblue/heartbeat/1/+json/example/default.js

```diff
@@ -0,0 +1,48 @@
+module.exports = {
+    _type: 'application/vnd.oada.isoblue.location.1+json',
```

Comment:
This file is the wrong name: you've given an example of a location stream (and the _type is location), but the file itself is under isoblue/heartbeat.  I think you've got the heartbeat and location mixed up

Reply:
Fixed in 5696ced